gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/op/AbstractOpProcessor.java

```diff
@@ -72,8 +73,19 @@ protected AbstractOpProcessor(final boolean manageTransactions) {
      * @param context The Gremlin Server {@link Context} object containing settings, request message, etc.
      * @param itty The result to iterator
      * @throws TimeoutException if the time taken to serialize the entire result set exceeds the allowable time.
+     * @see #handleIterator(ResponseHandlerContext, Iterator)
      */
     protected void handleIterator(final Context context, final Iterator itty) throws TimeoutException, InterruptedException {
+        handleIterator(new ResponseHandlerContext(context), itty);
+    }
+
+    /**
+     * A variant of {@link #handleIterator(Context, Iterator)} that is suitable for use in situations when multiple
+     * threads may produce {@link ResponseStatusCode#isFinalResponse() final} response messages concurrently.
```

Comment:
should `final` be moved out of the brackets?

Reply:
keeping current link syntax (as discussed in comments under `AbstractEvalOpProcessor.java`)